ipatests/test_integration/test_sudo.py

```diff
@@ -24,22 +24,43 @@
     clear_sssd_cache, get_host_ip_with_hostmask, modify_sssd_conf)
 
 
-class TestSudo(IntegrationTest):
-    """
-    Test Sudo
-    http://www.freeipa.org/page/V4/Sudo_Integration#Test_Plan
-    """
+class TestSudoBase(IntegrationTest):
```

Comment:
It's a good idea to *not* name a base class for tests as ``TestSomething``. pytest considers anything starting with ``test`` or ``Test`` as a test. For an abstract base class, we don't want pytest to discover the class. How about ``SudoBaseTest`` or ``BaseSudoTest`` instead?